contracts/cw721-stake/src/msg.rs

```diff
@@ -0,0 +1,105 @@
+use cosmwasm_std::Uint128;
+use cw721::Cw721ReceiveMsg;
+use cw_core_macros::voting_query;
+use cw_utils::Duration;
+use schemars::JsonSchema;
+use serde::{Deserialize, Serialize};
+
+pub use cw721_controllers::NftClaimsResponse;
+
+#[derive(Serialize, Deserialize, Clone, JsonSchema, Debug)]
+#[serde(rename_all = "snake_case")]
+pub enum Owner {
```

Comment:
is this strictly necessary considering i can just paste the contract address into the field?

Reply:
Intention here is to allow a DAO to instantiate this with itself as the owner during its own instantiation. If we just make this field an address you won't be able to put the address of the DAO that will be instantiated in the `ModuleInstantiateInfo`. Our current voting modules don't need this because the wrapper contract can handle the admin transfer.